client/src/main/java/com/paypal/selion/internal/platform/grid/browsercapabilities/AppiumCapabilitiesBuilder.java

```diff
@@ -44,14 +45,19 @@ public DesiredCapabilities getCapabilities(DesiredCapabilities capabilities) {
         }
         capabilities.setCapability(MOBILE_NODE_TYPE, mobileSession.getMobileNodeType().getAsString());
         capabilities.setCapability(MobileCapabilityType.DEVICE_NAME, mobileSession.getDeviceType());
+        if (mobileSession.getDeviceType() != null) {
```

Comment:
Oh sorry, it is actually a bug :-"
I should define a new parameter for setting browser name. Using device type as browser name will cause problems with appium on ios and on selenium grid with lots of nodes connected to it.
( it was a quick fix while developing that somehow got forgotten) :">